sky/packages/sky/lib/src/material/checkbox.dart

```diff
@@ -162,24 +162,25 @@ class _RenderCheckbox extends RenderToggleable {
       double t = (position - _kMidpoint) / (1.0 - _kMidpoint);
 
       // First draw a rounded rect outline then fill inner rectangle with accent color.
-      paint.color = new Color.fromARGB((t * 255).floor(), _accentColor.red,
-          _accentColor.green, _accentColor.blue);
-      paint.setStyle(ui.PaintingStyle.stroke);
+      paint
+        ..color = new Color.fromARGB((t * 255).floor(), _accentColor.red, _accentColor.green, _accentColor.blue)
+        ..style = ui.PaintingStyle.stroke;
       canvas.drawRRect(rrect, paint);
-      paint.setStyle(ui.PaintingStyle.fill);
+      paint.style = ui.PaintingStyle.fill;
       canvas.drawRect(innerRect, paint);
 
       // White inner check
-      paint.color = const ui.Color(0xFFFFFFFF);
-      paint.setStyle(ui.PaintingStyle.stroke);
-      ui.Path path = new ui.Path();
-      ui.Point start = new ui.Point(_kEdgeSize * 0.15, _kEdgeSize * 0.45);
-      ui.Point mid = new ui.Point(_kEdgeSize * 0.4, _kEdgeSize * 0.7);
-      ui.Point end = new ui.Point(_kEdgeSize * 0.85, _kEdgeSize * 0.25);
+      paint
+        ..color = const Color(0xFFFFFFFF)
+        ..style = ui.PaintingStyle.stroke;
+      Path path = new Path();
+      Point start = new Point(_kEdgeSize * 0.15, _kEdgeSize * 0.45);
+      Point mid = new Point(_kEdgeSize * 0.4, _kEdgeSize * 0.7);
+      Point end = new Point(_kEdgeSize * 0.85, _kEdgeSize * 0.25);
       Point lerp(Point p1, Point p2, double t) =>
```

Comment:
Don't we have a Point.lerp? Also, this line is quite confusing in the middle of all this other imperative code. Also, style guide says not to split => over two lines.